torch/_tensor_docs.py

```diff
@@ -4845,6 +4845,49 @@ def callable(a, b) -> number
 ``x.T`` is equivalent to ``x.permute(n-1, n-2, ..., 0)``.
 """)
 
+add_docstr_all('H',
+               r"""
+Returns a view of a matrix (2-D tensor) conjugated and transposed.
+
+``x.H`` is equivalent to ``x.transpose(0, 1).conj()`` for complex matrices and
+``x.transpose(0, 1)`` for real matrices.
+
+.. seealso::
+
+        :attr:`~.Tensor.mH`: An attribute that also works on batches of matrices.
+""")
+
+add_docstr_all('mT',
+               r"""
+Is this tensor with the last two dimensions transposed.
+
+``x.mT`` is equivalent to ``x.transpose(-2, -1)``.
+""")
+
+add_docstr_all('mH',
+               r"""
+Alias for :meth:`~Tensor.adjoint`.
+""")
+
+add_docstr_all('adjoint',
+               r"""
+adjoint(Tensor) -> Tensor
+Returns a view of the tensor conjugated and with the last two dimensions transposed.
+
+``x.adjoint()`` is equivalent to ``x.transpose(-2, -1).conj()`` for complex tensors and
+``x.transpose(-2, -1)`` for real tensors.
+
+Example::
+    >>> x = torch.arange(4, dtype=torch.float)
+    >>> A = torch.complex(x, x.flip(0)).reshape(2, 2)
```

Comment:
starting with a floating point tensor here seems a little circuitous but it's probably fine; might be easier to just construct the complex tensor by hand to being so people don't think the flipping and reshaping are relevant to what the op does

Reply:
I liked this idea so that it's reproducible. This operation is just moving stuff around and negating some parts. I liked a simple example because it's easier to follow what goes where. I've simplified it a bit more even, tell me what you think.